### PR TITLE
Removed deprecated CSS-based positioning from Tooltip

### PR DESCRIPTION
**Pull request by conor-cafferkey-sociomantic**
Closes https://github.com/sociomantic-tsunami/nessie-ui/issues/402:
- Removes support for CSS-positioned tooltips
- Tooltip is _just the tooltip_
- `children` is now the tooltip content

### CODE REVIEW

**Comment by daniel-martic-sociomantic** [comment]
There's one thing: at the moment, Tooltip is always displayed in other components that are using it (e.g. `ComboBox`). Maybe we should keep `isVisible` in order to properly show/hide `Tooltip` in other components; what do you think?

**Comment by conor-cafferkey-sociomantic** [comment]
Thanks for catching @daniel-martic-sociomantic! I’ve removed the Tooltip from any other components now.

**Comment by daniel-martic-sociomantic** [comment]
Nice! One little thing left 😁 : since `position` prop is handling where to place arrow (the little triangle) around Tooltip, maybe we should rename the prop to something like `arrowPosition` or something similar… Since the values of this prop are actually opposite from the arrow position (because previously they reflected the actual positioning of Tooltip), maybe we should also update the values and CSS class names to reflect this (i.e. `position: top` will place this arrow on bottom; `position: left` on right etc.).

**Comment by conor-cafferkey-sociomantic** [comment]
Good point! Updated ☝️

File: src/Tooltip/README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-CSS. This approach is deprecated. You should position the tooltip using
-a library such as [Popper.js](https://popper.js.org/) instead.**
+CSS. This is no longer supported. You should position the tooltip using a
+library such as [Popper.js](https://popper.js.org/) instead.**
 
 ## Example Usage
 
 ```
-<Tooltip message="There’s an error!" position="top"/>
+<Tooltip message="There’s an error!" position="top" />
```
[the remark this answers is not in the document]
Comment (conor-cafferkey-sociomantic) [reply src/Tooltip/README.md]: Thanks for catching!

**Pull request merged**